research/filters/procedure_filter.py

```diff
@@ -0,0 +1,37 @@
+"""
+Definition of the :class:`ProcedureFilter` class.
+"""
+
+from django_filters import rest_framework as filters
+from research.models.procedure import Procedure
+from research.models.study import Study
+
+
+class ProcedureFilter(filters.FilterSet):
+    """
+    Provides useful filtering options for the
+    :class:`~research.models.procedure.Procedure` model.
+
+    """
+
+    title = filters.LookupChoiceFilter(
+        lookup_choices=[
+            ("contains", "Contains (case-sensitive)"),
+            ("icontains", "Contains (case-insensitive)"),
+            ("exact", "Exact"),
+        ]
+    )
+    description = filters.LookupChoiceFilter(
+        lookup_choices=[
+            ("contains", "Contains (case-sensitive)"),
+            ("icontains", "Contains (case-insensitive)"),
+            ("exact", "Exact"),
+        ]
+    )
```

Comment:
Why not use the `LOOKUP_CHOICES` from `research.filters.utils`?

Reply:
No good reason, didn't have the time to refactor. I'll merge to try and set up things on the server and then we can make improvements.